### ISSUE TITLE
Issues with HA 0.106.x?

### ISSUE THREAD

**Issue by gurbyz** [gurbyz/power-wheel-card#37]
Please see closed issues #34, #35 and #36 for hints. You probably are impacted by caching or a HACS issue, not by an issue in power-wheel-card.

**Comment by atze09** [comment]
> Please see closed issues #34, #35 and #36 for hints. You probably are impacted by caching or a HACS issue, not by an issue in power-wheel-card.

I solved it for my browser, but not for the home assistant android app.
There is still the failure:
"connot add property production_is_positive, object is not extensible"
Could this also be solved?

**Comment by gurbyz** [comment]
I think that's a caching issue. Maybe you can force your app to update the file by reopening the app or that kind of solutions. Or maybe you can solve it by 'saying to your app' that the file has changed by changing the `v=<number>` in the line below:
```
resources:
  - url: /local/power-wheel-card.js?v=1
    type: module
```
Your url could be different, e.g. when you install via HACS. If you don't have the `?v=1`, just add that. If you have a number, add 1 to the value. And restart HA.
